src/markup.js

```diff
@@ -10,7 +10,7 @@ ${getBadge(jsonResults.stats.requests, 'Requests')}
 ${getBadge(jsonResults.stats.assertions, 'Assertions')}
 ${getTestTimes(jsonResults.timings)}
 ${getTestCounters(jsonResults)}
-${getTestResultsMarkup(jsonResults.failures)}
+${getTestResultsMarkup(jsonResults.failures, reportName)}
```

Comment:
This function call was missing the `reportName`, so `undefined` would appear in Github comments.